WhlgPortalWebsite.UnitTests/Website/Controllers/ServiceManagerControllerTests.cs

```diff
@@ -0,0 +1,87 @@
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using WhlgPortalWebsite.BusinessLogic.Services;
+using WhlgPortalWebsite.Controllers;
+using WhlgPortalWebsite.Models;
+
+namespace Tests.Website.Controllers;
+
+[TestFixture]
+public class ServiceManagerControllerTests
+{
+    [Test]
+    public async Task OnboardDeliveryPartnerAsync_ShouldRedirectWhenEmailIsValid()
+    {
+        // Arrange
+        var mockUserService = new Mock<IUserService>();
+        mockUserService
+            .Setup(x => x.IsEmailAddressInUseAsync("existing@email.com"))
+            .ReturnsAsync(false);
```

Comment:
It would be nice to have this test add a user to the DB instead of mocking this function out.
Currently these tests don't test the functionality of IsEmailInUseAsync - this would be nice to have in a DataAccessProvider test suite but perhaps that's a future testing improvement